mysql-test/suite/galera/t/galera_sst_xtrabackup-v2_privileges.cnf

```diff
@@ -0,0 +1,9 @@
+!include ../galera_2nodes.cnf
+
+[mysqld]
+wsrep_sst_method=xtrabackup-v2
+wsrep_debug=1
+
+[xtrabackup]
+redo-log-arch-dir='backup1:/tmp/mysql-redo-archive/'
```

Comment:
Doesn't this create some files in `/tmp/mysql-redo-archive`?

Reply:
fixed